src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -541,46 +463,20 @@ export default {
 				);
 			}
 		},
-		initializeRecommendedRowAlgoExp() {
-			// experiment: VUE-937 for "recommend by others" row backing algorithm
-			const experimentId = 'EXP-VUE-937-recommended-row-algo';
-
-			// get experiment assignment
-			const { version } = this.apollo.readFragment({
-				id: `Experiment:${experimentId}`,
-				fragment: experimentVersionFragment,
-			}) || {};
-
-			// track version if it is set
-			if (version && version !== 'unassigned') {
-				this.$kvTrackEvent('Lending', experimentId, version);
-			}
-		},
 	},
 	apollo: {
 		preFetch(config, client) {
 			let rowData;
 			let expRowData;
 			let expResults;
-
 			return client.query({
 				query: lendByCategoryQuery
 			}).then(({ data }) => {
 				// Get the array of channel objects from settings
 				rowData = readJSONSetting(data, 'general.rows.value') || [];
 				return Promise.all([
 					// experiment: GROW-330 Machine Learning Category row
-					client.query({ query: experimentQuery, variables: { id: 'EXP-ML-Service-Bandit-LendByCategory' } }),
```

Comment:
I think we need to keep this one

Reply:
You're right! I was keeping `EXP-VUE-937-recommended-row-algo` experiment. it's fixed now!